src/style.css

```diff
@@ -0,0 +1,31 @@
+
+* {
```

Comment:
do not use `*` to reset margins
it will reset margins for all elements, it is not good for performance